example-apps/kitchen-sink-new-router/source/components/metadata-test/metadata-test.ts

```diff
@@ -0,0 +1,59 @@
+import {Component, Input, Output} from '@angular/core';
+
+@Component({
+    selector: "attr-from-array",
+    inputs: ["attrIn"],
+    outputs: ["attrOut"],
+    template: `<p>{{attrIn}}</p>`
+})
+export class MetadataFromArray {
+}
+
+@Component({
+    selector: "attr-from-decorator",
+    template: `<p>{{attrIn}}</p>`
+})
+export class MetadataFromDecorator {
+    @Input() attrIn: any;
+    @Output() attrOut: any;
+}
+
+
+@Component({
+    selector: "attr-from-array-custom",
+    inputs: ["attrIn:attrCustomIn"],
+    outputs: ["attrOut"],
+    template: `<p>{{attrIn}}</p>`
+})
+export class MetadataFromArrayWithCustomName {
+}
+
+@Component({
+    selector: "attr-from-decorator-custom",
+    template: `<p>{{attrIn}}</p>`
+})
+export class MetadataFromDecoratorWithCustomName {
+    @Input("attrCustomIn") attrIn: any;
+    @Output() attrOut: any;
+}
+
+
+@Component({
+  selector: 'metadata-test',
```

Comment:
The purpose of this test it to verify the accordion on the side panel in Augury, whenever the inputs array is present of the inline `@Input/@Output`. Along with tests that verify custom names on inputs (@Input('attrCustomIn')) or `inputs: ['attrIn:attrCustomIn']`